CMakeLists.txt

```diff
@@ -2,8 +2,27 @@ cmake_minimum_required(VERSION 2.8)
 
 project(GPUI)
 
-set(GPUI_VERSION_MAJOR 0)
-set(GPUI_VERSION_MINOR 1)
+execute_process(
+    COMMAND
+    bash -c "grep 'Version:' .gear/gpui.spec | awk -F ':' '{ print $2 }' | tr -d [:space:] | awk -F '.' '{ print $1 }'"
```

Comment:
While it is perfectly fine to launch bash script three times in a row, I would like you to avoid that. Please utilize cmake to do so, for example:
```
cmake_minimum_required(VERSION 3.12)

set(MY_PROGRAM_VERSION "0.2.17")

string(REPLACE "." ";" VERSION_LIST ${MY_PROGRAM_VERSION})
list(GET VERSION_LIST 0 MY_PROGRAM_VERSION_MAJOR)
list(GET VERSION_LIST 1 MY_PROGRAM_VERSION_MINOR)
list(GET VERSION_LIST 2 MY_PROGRAM_VERSION_PATCH)
```
Code above translates version string to list and uses access by index operation to extract necessary parts.